File: src/main/java/test1/Hello.java
package test1;

public class Hello {
    public static void main(String[] args) {  //psvm 設計程式main
//        System.out.println("Hello World");
        //new Persion().hello();  //將類別產生在記憶體
        Persion p = new Persion(); //物件Persion中建立p來代表Persion物件
        p.hello();

        int a = 1;
        Integer a_class = 1;  //int的類別就能有其他function
        a_class.byteValue();  //有功能性的資料稱為物件＆參照資料型態

        float b = 12.2f; //預設都為double
        Float bb = 12f;  //每個基本資料型態都有包裝類別wrapper class
        double c = 1.2;
        Double cc =1.2;
        boolean adult = true;

        char aa = 'C';
        byte aaa = 'C';
        String name = "Tom"; //字串


    }



}

File: src/main/java/test1/Hello.kt
package test1

fun main() {        //設計function
//    println("Hello kotlin");
//    Human().hello()
    val h=Human()  //val定義完這物件就不能改變值   //var變數可以改變
    h.hello()

    val age: Int = 20;  // val 名稱：變數＝值
    var weigh = 60.5; // kolin會自行選擇資料型態（double)

    var name:String
    name = "Tom"


}

class Human{        //設計類別
    fun hello(){    //回傳值不需寫在function前面(void)
        println("Hello Kotlin")

    }


}